1.7/usage/managing-services/creating.md

```diff
@@ -0,0 +1,222 @@
+
+Packages are published to the Mesosphere Universe repository on GitHub. At their simplest, DC/OS packages are bundled containerized apps that can be deployed onto a cluster. 
+
+## Getting Started
+
+1.  Install these prerequisite tools.
+    
+    -  Install the DC/OS and the DC/OS CLI using this [guide](/docs/1.7/administration/installing/). You must have at least 1 [public agent](https://docs.mesosphere.com/1.7/overview/concepts/#public) node.
+    -  Install [pip](https://pip.pypa.io/en/stable/installing.html#install-pip)
+    -  Install jsonschema:
+    
+       ```bash
+       $ pip install jsonschema
+       ```
+       
+1.  Fork the [Universe](https://github.com/mesosphere/universe) repository, then clone the fork:
+
+    ```bash
+    $ git clone https://github.com/<your-username>/universe
+    ```
+    
+    As you look around the Universe repository you'll notice that the repository (`/repo/packages`) consists of packages arranged alphabetically. Each package consists of 4 JSON files:
+    
+    -  `config.json` -  configuration properties supported by the package, represented as a json-schema.
+    -  `package.json` -  high-level metadata about the package.
+    -  `marathon.json.mustache` - a [mustache](http://mustache.github.io/) template that when rendered will create a [Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service.
+    -  `resource.json` - contains all of the externally hosted resources (e.g. Docker images, HTTP objects and images) that are required to install the application.
+
+1.  Install the Universe repository pre-commit hooks.
+
+    ```bash
+    $ bash scripts/install-git-hooks.sh
+    ```
+
+1.  Create a new directory for the your package in `repo/packages/`. For example, to create a package named `helloworld` the structure is `repo/packages/H/helloworld/0`. The `0` directory corresponds to the release number of the package.  If you look at other packages in Universe, many have multiple releases (directories `0`, `1`, `2`, etc).
+
+Now, you're ready to start creating your package!  
+
+## Step One: Create package.json
+
+Every package in Universe must have a package.json file which specifies the high level metadata about the package. This file specifies the highest-level metadata about the package (comparable to a `package.json` in Node.js or `setup.py` in Python).
+
+Currently, a package can specify one of two values for `.packagingVersion`, either 2.0 or 3.0. The version is declared will dictate which other files are required for the complete package as well as the schemas all the files must adhere to. 
+
+Below is a snippet that represents a version 2.0 package. See [`repo/meta/schema/package-schema.json`](https://github.com/mesosphere/universe/blob/version-3.x/repo/meta/schema/package-schema.json) for the full JSON schema outlining what properties are available for each corresponding version of a package.
```

Comment:
Instead of pointing to the raw schema files, let's point them to the universe readme with the higher level summary of things: https://github.com/mesosphere/universe#packagejson